store/cassandra/cassandra.go

```diff
@@ -433,17 +433,14 @@ func (c *CassandraStore) SearchTable(ctx context.Context, key schema.AMKey, tabl
 
 	pre := time.Now()
 
-	start_month := start - (start % Month_sec)       // starting row has to be at, or before, requested start
-	end_month := (end - 1) - ((end - 1) % Month_sec) // ending row has to include the last point we might need (end-1)
-
 	// unfortunately in the database we only have the t0's of all chunks.
 	// this means we can easily make sure to include the correct last chunk (just query for a t0 < end, the last chunk will contain the last needed data)
 	// but it becomes hard to find which should be the first chunk to include. we can't just query for start <= t0 because than we will miss some data at
 	// the beginning. We can't assume we know the chunkSpan so we can't just calculate the t0 >= (start - <some-predefined-number>) because chunkSpans
 	// may change over time.
 	// we effectively need all chunks with a t0 > start, as well as the last chunk with a t0 <= start.
```

Comment:
shouldn't this be `as well as the last chunk with a t0 <= end` instead of `start`

Reply:
no